portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -794,6 +794,13 @@ definition {
 			value1 = "${fieldValue}");
 	}
 
+	macro viewFieldLabelByDataFieldName {
+		AssertElementPresent(
+			fieldLabel = "${fieldLabel}",
```

Comment:
@fmarcio, this is not using the pattern used by the other macros, can you please update according to the example below?
```
key_fieldLabel = "${fieldLabel}",
```

Reply:
> @fmarcio, this is not using the pattern used by the other macros, can you please update according to the example below?
> 
> ```
> key_fieldLabel = "${fieldLabel}",
> ```

@natocesarrego fixed! :-)

Reply:
@fmarcio, the new macro's code is:
```
var key_fieldName = "${key_fieldName}";
var key_fieldLabel = "${key_fieldLabel}";
```
However, the expected pattern is:
```
var key_fieldName = "${fieldName}";
var key_fieldLabel = "${fieldLabel}";
```
Can you please fix?

Thanks.

Reply:
> @fmarcio, the new macro's code is:
> 
> ```
> var key_fieldName = "${key_fieldName}";
> var key_fieldLabel = "${key_fieldLabel}";
> ```
> 
> However, the expected pattern is:
> 
> ```
> var key_fieldName = "${fieldName}";
> var key_fieldLabel = "${fieldLabel}";
> ```
> 
> Can you please fix?
> 
> Thanks.

@natocesarrego fixed! :-)